terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -762,6 +841,11 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                 "security_groups": [
                     "${aws_security_group.gitlab_alb.id}"
                 ],
+                "access_logs": {
+                    "bucket": "${aws_s3_bucket.gitlab_logs.id}",
+                    "prefix": gitlab_logs_path.rsplit('/', 3)[0],
```

Comment:
The point of extracting a variable is so that it can be freely changed. This looks like it needs to be adjusted when the number of slashes in the variable value changes.